tests/pages/bigtest/package-show.js

```diff
@@ -0,0 +1,23 @@
+import {
+  clickable,
+  isPresent,
+  page,
+  property,
+} from '@bigtest/interaction';
+
+@page class PackageShowPage {
+  allowKbToAddTitles = property('checked', '[data-test-eholdings-package-details-allow-add-new-titles] input');
+  hasAllowKbToAddTitles = isPresent('[data-test-eholdings-package-details-toggle-allow-add-new-titles] input');
+  hasAllowKbToAddTitlesToggle = isPresent('[package-details-toggle-allow-add-new-titles-switch]');
+  isSelected = property('checked', '[data-test-eholdings-package-details-selected] input');
+  toggleAllowKbToAddTitles = clickable('[data-test-eholdings-package-details-allow-add-new-titles] input');
+  toggleIsSelected = clickable('[data-test-eholdings-package-details-selected] input');
+}
+
+@page class PackageShowModalPage {
```

Comment:
I actually just thought of something.

Instead of exporting a named page object for the modal, we could add a property to `PackageShowPage` that is the modal.

```
@page class PackageShowPage {
  modal = new PackageShowModal('#eholdings-package-confirmation-modal')
}
```

And in the test, we can do

```
PackageShowPage.toggleIsSelected()
  .append(PackageShowPage.modal.confirmSelection())
```

Reply:
That's a good idea, let me give that  a shot.